ruby/.rubocop.yml

```diff
@@ -0,0 +1,8 @@
+# Commonly used screens these days easily fit more than 80 characters.
+Metrics/LineLength:
+  Max: 120
+
+# Too short methods lead to extraction of single-use methods, which can make
+# the code easier to read (by naming things), but can also clutter the class
+Metrics/MethodLength: 
+  Max: 20
```

Comment:
20 is a really generous number for the method length :D

Please remember that we are trying to make single-use methods a habit and rubocop enforces it.
In addition, all the ruby/rails projects are not overcomplicated projects with complex logic.

Let's be less strict with a controller though 🤔 
Can you change it to 🙏  :

```
Metrics/MethodLength:
  Include: ["app/controllers/*"]
  Max: 20
```